MavenQuickStart/quickApp/src/main/java/com/globant/topictwo/exercisetwo/Blog.java

```diff
@@ -0,0 +1,145 @@
+package com.globant.topictwo.exercisetwo;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import org.apache.log4j.Logger;
+
+/**
+ * This class represents the features and behavior of our blog, and use Entry to
+ * post new information.
+ * 
+ * @author andres.vaninetti
+ *
+ */
+public class Blog {
+
+	private static final int PAGINATION = 10;
+	private static final Logger LOGGER = Logger.getLogger(Blog.class.getName());
+
+	private String name;
+	private String owner;
+	private String theme;
+	private List<Entry> entries;
+
+	/**
+	 * Deafult constructor.
+	 * 
+	 * @author andres.vaninetti
+	 *
+	 */
+	public Blog() {
+
+	}
+
+	/**
+	 * Class constructor with parameters.
+	 * 
+	 * @param name
+	 * @param owner
+	 * @param theme
+	 * @param entries
+	 */
+	public Blog(String name, String owner, String theme, LinkedList<Entry> entries) {
```

Comment:
Is better to use the interface instead of the implementor, entries should be a List